project/Dependencies.scala

```diff
@@ -9,36 +9,45 @@ object Dependencies {
-    "io.spray"            %%  "spray-json"    % "1.3.2",
-    "io.spray"            %%  "spray-client"  % sprayV,
-    "io.spray"            %%  "spray-testkit" % sprayV    % "test",
-    "org.webjars"          %  "swagger-ui"    % "2.2.5",
-    "com.typesafe.akka"   %%  "akka-actor"    % akkaV,
-    "com.typesafe.akka"   %%  "akka-contrib"  % akkaV,
-    "com.typesafe.akka"   %%  "akka-testkit"  % akkaV     % "test",
-    "com.typesafe.akka"   %%  "akka-slf4j"    % akkaV,
-    "org.elasticsearch"    % "elasticsearch"  % "2.4.1",
-    ("com.google.api-client" % "google-api-client" % "1.22.0").exclude("com.google.guava", "guava-jdk5"),
-    "com.google.apis" % "google-api-services-storage" % "v1-rev58-1.21.0",
-    "com.google.apis" % "google-api-services-compute" % "v1-rev120-1.22.0",
-    "com.jason-goodwin"   %% "authentikat-jwt" % "0.4.1",
-    "com.sun.mail"         % "javax.mail" % "1.5.6",
-    "com.univocity"        % "univocity-parsers" % "2.4.1",
-    "org.ocpsoft.prettytime" % "prettytime" % "4.0.1.Final",
-    "org.everit.json"      %  "org.everit.json.schema" % "1.4.1",
-    "org.specs2"          %%  "specs2-core"   % "3.7"  % "test",
-    "org.scalatest"       %%  "scalatest"     % "2.2.6"   % "test",
-    "org.mock-server"      %  "mockserver-netty" % "3.10.2" % "test"
+
+    "io.spray"                      %% "spray-can"           % sprayV,
+    "io.spray"                      %% "spray-routing"       % sprayV,
+    "io.spray"                      %% "spray-json"          % "1.3.2",
+    "io.spray"                      %% "spray-client"        % sprayV,
+    "io.spray"                      %% "spray-testkit"       % sprayV    % "test",
+    "com.typesafe.akka"             %% "akka-actor"          % akkaV,
+    "com.typesafe.akka"             %% "akka-testkit"        % akkaV     % "test",
+    "com.typesafe.akka"             %% "akka-slf4j"          % akkaV,
+
+    "org.elasticsearch.client"       % "transport"           % "5.3.0",
+
+    "com.google.api-client"          % "google-api-client"   % "1.22.0"
+      exclude("com.google.guava", "guava-jdk5"),
+    "com.google.apis"                % "google-api-services-storage" % "v1-rev58-1.21.0",
+    "com.google.apis"                % "google-api-services-compute" % "v1-rev120-1.22.0",
+
+    "org.webjars"                    % "swagger-ui"          % "2.2.5",
+    "com.jason-goodwin"             %% "authentikat-jwt"     % "0.4.1",
+    "com.sun.mail"                   % "javax.mail"          % "1.5.6",
+    "com.univocity"                  % "univocity-parsers"   % "2.4.1",
+    "org.ocpsoft.prettytime"         % "prettytime"          % "4.0.1.Final",
+    "org.everit.json"                % "org.everit.json.schema" % "1.4.1",
+
+    "org.specs2"                    %% "specs2-core"         % "3.7"     % "test",
+    "org.scalatest"                 %% "scalatest"           % "2.2.6"   % "test",
+    "org.mock-server"                % "mockserver-netty"    % "3.10.2"  % "test"
```

Comment:
this is mostly whitespace/formatting changes - try the `?w=1` github trick to ignore whitespace!